app/assets/javascripts/proposals.js

```diff
@@ -0,0 +1,26 @@
+!function() {
+  function bindProposal(form) {
+    var form = $(form)
+      , pointsContainer = form.find('.proposal-points')
```

Comment:
Comma warnings can be turned off with 'laxcomma'.<br>Bad line breaking before ','.